### ISSUE TITLE
[wasi] Failing test `System.Globalization.Tests.CalendarTests.TestJapaneseCalendarDateParsing`

### ISSUE THREAD

**Issue by radical** [dotnet/runtime#82965]
Library: System.Globalization.CalendarsWithConfigSwitch.Tests

```
  fail: [FAIL] System.Globalization.Tests.CalendarTests.TestJapaneseCalendarDateParsing
  info: Assert.Throws() Failure
  info: Expected: typeof(System.FormatException)
  info: Actual:   (No exception was thrown)
  info:    at System.Globalization.Tests.CalendarTests.TestJapaneseCalendarDateParsing()
  info:    at System.Reflection.MethodInvoker.InterpretedInvoke(Object obj, IntPtr* args)
  info:    at System.Reflection.MethodInvoker.Invoke(Object obj, IntPtr* args, BindingFlags invokeAttr)
```

cc @ilonatommy @mkhamoyan

**Comment by radical** [comment]
Should this whole thing be disabled like the following? :

https://github.com/dotnet/runtime/blob/732ae12c9cdd8227ab2882f0949d798c3fcb446d/src/libraries/tests.proj#L345-L347

**Comment by ilonatommy** [comment]
There might be 2 reasones why we are not throwing: 
1) data was found correctly (I don't think so),
2) `predefined cultures only` is set to false and in the test we did not indicate it should not be.
It's rather the latter option and most probably we don't expect the test to inish with exception, it might have been just a copy-paste test before @pavelsavara's wasm-blazor startup unification where we stopped setting `predefined cultures only` to true for all non-blazor applications. I will need to look into it to confirm and let you know. If it's urgent, then blocking the test is for sure fine.
